pkg/operator/ceph/cluster/watcher.go

```diff
@@ -38,6 +38,8 @@ type clientCluster struct {
 	context   *clusterd.Context
 }
 
+var alreadyPresentOSDs = make(map[string]string)
```

Comment:
How about this name? And rook's `util.Set` type has some convenient helpers for Add(), Contains(), etc that will make it more natural to check for existence.
```suggestion
var nodesCheckedForReconcile = util.NewSet()
```